app/views/settings/identity_proofs/new.html.haml

```diff
@@ -27,5 +27,10 @@
 
           %p= t('identity_proofs.i_am_html', username: content_tag(:strong, @proof.provider_username), service: @proof.provider.capitalize)
 
+    .connection-prompt__post
+      = f.input :post_status, label: "And toot this:", as: :boolean, wrapper: :with_label, :input_html => { checked: true, value: 'true' }
```

Comment:
Not localized, and it looks like the checkbox does not align well with the label for some reason (in the screenshot)...